### PR TITLE
Adding autocomplete to the rally page (maybe?)

### PR DESCRIPTION
**Pull request by jontg**
I'm not sure if this feature makes sense here, but if we are `online` then it could be useful to have this.  If we flip to `offline` mode, what's the best way to kill off autocomplete without OOM-whispering.

### CODE REVIEW

File: static/rally/index.html [hunk 1]
```diff
@@ -189,5 +189,22 @@
...
 <!-- End Google Analytics -->
 
+<script>
+  var autocomplete;
+
+  function initAutocomplete() {
+    // Create the autocomplete object, restricting the search to geographical
+    // location types.
+    autocomplete = new google.maps.places.Autocomplete(
+        /** @type {!HTMLInputElement} */(document.getElementById('address_autocomplete')),
```
Comment (xavdid) [inline static/rally/index.html:199]: this is unusual syntax. there's an inline type declaration? 

Also is there a reason for the extra set of parens around `getElementbyId`?

**Review by xavdid — APPROVED** [review]
looks good, just have a minor style question!

File: static/rally/index.html (hunk 1 above)
Comment (jontg) [reply static/rally/index.html:199]: yeah, it's actually stolen wholesale from the google maps autocomplete example — it seems to work, so the not breaking what isn't broke rule is what I followed ;-)

Comment (xavdid) [reply static/rally/index.html:199]: 👍 😃

**Pull request merged**